src/utils/GetChunkSize.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+namespace chunk_size {
+
+/**
+ * Get the chunk size based on the given magic numbers.
+ * @param loop_size The number of iterations.
+ * @param max_num_chunks should be big enough to provide sufficient scheduling, but not too big, as too much overhead
+ * might exist.
+ * @param max_chunk_size should not be bigger than needed. When iterating the specified number of work items, the
+ * overhead should be negligible. Bigger numbers should not produce any better results.
```

Comment:
```suggestion
 * @param max_chunk_size When iterating the specified number of work items, the overhead should be negligible. 
 *  Bigger numbers should not produce any better results.
```
Actually, I still think this does not really say anything.